modules/administration/pages/custom-ssl.adoc

```diff
@@ -194,3 +205,85 @@ Click btn:[Find Targets].
 . Check that the appropriate clients are displayed, and click btn:[Run Command] to deploy the certificates on the clients.
 
 image::deploy_cert_salt.png[scaledwidth=80%]
+
+When you have every client trusting both the old and new certificates, you can go ahead and replace the server certificate on the {productname} Server and Proxies.
+
+
+
+.Procedure: Replace Server Certificate on the Server
+
+. On the {productname} Server, at the command prompt, install the RPM from the [path]``ssl-build`` directory:
++
+----
+rpm -Uhv ssl-build/susemanager/rhn-org-httpd-ssl-key-pair-susemanager-1.0-2.noarch.rpm
+----
+. Restart services to pick the changes:
++
+----
+spacewalk-service restart
+----
+
+
+
+.Procedure: Replace Server Certificate on the Proxy
+
+. On the {productname} Proxy, at the command prompt, install the RPM from the [path]``ssl-build`` directory:
++
+----
+rpm -Uhv ssl-build/susemanager-proxy/rhn-org-httpd-ssl-key-pair-susemanager-proxy-1.0-2.noarch.rpm
+----
+. Restart services to pick up the changes:
++
+----
+spacewalk-service restart
+----
+
+
+When you have replaced the server certificates on your server and any proxies, you need to update the certificate with only the new details all the clients.
```

Comment:
Maybe, "_on_ all the clients".

Reply:
Maybe we should add here, that the customer should "test" if the clients still operate correctly and can reach the server/proxies via SSL.